### PR TITLE
UofT-DSI | Applying Statistical Concepts - Assignment 1 *updated*

### PR DESCRIPTION
**Pull request by KathrynVozoris**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Adding solutions to assignment 1.

## What did you learn from the changes you have made?
I learned how to standardize and split the data. I learned more about working with dataframes, and navigating their rows and columns. I learned how to use cross validation, and knn models. I learned how to choose the best K for the n_neighbors parameter, and check the accuracy of the model on the test data.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
I had some trouble with splitting the data, due to the fact that the predictor variables had been standardized already, and this was different than the examples we had looked at. The train_test_split function requires the full dataframe that you started with, so I had to join the predictors_standardized with the ['class'] column to get the full data set in standardized form.

## How were these changes tested?
I looked at the results in the dataframes, by printing them, and looked at graphs of the results as well.

## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

**Comment by KathrynVozoris** [comment]
I had called the branch 'assignments' instead of assignment-1, so had to redo the pull request

**Review by amanda-ng518 — APPROVED** [review]
A1: PASS
you should get 8 for Q3 and ~0.947 for Q4

**Comment by KathrynVozoris** [comment]
I had gotten the results you quoted. Somehow my last pushes to the repo did not work due to conflicts in the (.../03_instrcuctional_team/markdown_slides/images...). I was still unable to stage the changes and had to just push them directly.
